apps/ejabberd/src/mod_commands.erl

```diff
@@ -107,25 +107,27 @@ commands() ->
       {result, ok}
      ],
      [
-      {name, get_last_messages},
-      {category, <<"messages">>},
-      {desc, <<"Get n last messages">>},
-      {module, ?MODULE},
-      {function, get_recent_messages},
-      {action, read},
-      {security_policy, [user]},
-      {args, [{caller, binary}, {other, binary}, {limit, integer}]},
-      {result, []}
+     {name, get_last_messages_with_everybody},
+     {category, <<"messages">>},
+     {desc, <<"Get n last messages with given contact, optionally before a certain date (unixtime)">>},
```

Comment:
The desc says this give last messages with given contact, but the name suggests it will return messages with everybody. Which one is correct?